tutorials/debugging/vscode-native/README.md

```diff
@@ -0,0 +1,268 @@
+
+   ```json
+   "sourceFileMap": {
+        "/workspace/holoscan-sdk/src/": "/home/user/holoscan-sdk/src"
+    },
+   ```
+
+   > 💡 Tip: Change `/home/user/holoscan-sdk` to the path where you cloned the Holoscan SDK repository.
+
+3. Clean up the existing build directory by opening the **Command Palette** and select **CMake: Clean Rebuild** and select **build**.
+
+4. Add Holoscan SDK source code to the workspace: Select **Add Folder to Workspace...** from the **File** menu. Find and select the directory where Holoscan SDK repository resides. 
+   The **Explorer** should now look like this:
+   <figure>
+   <img src="static/vscode-cpp-holoscan-sdk.png" alt="VS Code C++ Workspace with Holoscan SDK" />
+   <figcaption>Figure 4: VS Code workspace with Holoscan SDK.</figcaption>
+   </figure>
+
+5. Open `src/core/application.cpp` from the `holoscan-sdk` folder. Find `void Application::run()` and set a breakpoint inside the function.
+
+6. Switch to the **Run and Debug** tab in VS Code. Select **(gdb) Launch Hello World** from the dropdown and click **Start Debugging** icon (play button ▶️) to start debugging. This time, VS Code should stop in the `main()` of Hello World and inside `void Application::run()`.
+
+#### Testing
+
+The C++ Hello World sample application includes a test case in the [CMakeLists.txt](./cpp/src/CMakeLists.txt#L31) file. To run this test case, ensure the **Test Preset** is set to `[Test All]` and then click the **Run CTest** button on the status bar. This action should run the test case and output the following:
+
+```bash
+[main] Building folder: /.../holohub/tutorials/debugging/vscode-native/cpp/build/debug
+[build] Starting build
+[proc] Executing command: /usr/local/bin/cmake --build /.../holohub/tutorials/debugging/vscode-native/cpp/build/debug --config Debug --target hello_world --
+[build] ninja: no work to do.
+[driver] Build completed: 00:00:00.027
+[build] Build finished with exit code 0
+[proc] Executing command: /usr/local/bin/ctest -T test --output-on-failure --output-on-failure --stop-on-failure -R ^EXAMPLE_CPP_HELLO_WORLD_TEST$
+[ctest]    Site: holoscan-dev
+[ctest]    Build name: Linux-g++
+[ctest] Test project /.../holohub/tutorials/debugging/vscode-native/cpp/build/debug
+[ctest]     Start 1: EXAMPLE_CPP_HELLO_WORLD_TEST
+[ctest] 1/1 Test #1: EXAMPLE_CPP_HELLO_WORLD_TEST .....   Passed    0.03 sec
+[ctest]
+[ctest] 100% tests passed, 0 tests failed out of 1
+[ctest]
+[ctest] Total Test time (real) =   0.03 sec
+[ctest] CTest finished with return code 0
+```
+
+### Python
+
+The Python example resides in `tutorials/debugging/vscode-native/python`. Open it with VS Code with the following command and install the dependencies:
+
+```bash
+code tutorials/debugging/vscode-native/python
+
+# Configure a Python virtual environment if desired, e.g.,
+python3 -m venv .venv
+
+# Install dependencies
+pip install -r ./requirements.txt
+```
```

Comment:
Should a user run the latter commands in their standalone shell or in a VSCode terminal? Or does it not matter?

Reply:
Great point! Let me break it up into two parts.